ouroboros-consensus/src/Ouroboros/Storage/ChainDB/Impl/Types.hs

```diff
@@ -325,8 +326,10 @@ data TraceOpenEvent blk
 
 -- | Trace type for the various events that occur when adding a block.
 data TraceAddBlockEvent blk
-  = AddedBlockToVolDB    (Point blk)
+  = AddedBlockToVolDB    !(Point blk) !BlockNo !Bool
```

Comment:
I don't like this naked Boolean. Is there a pre-existing `IsEBB`-ish data type?

Reply:
Not that I know of. There is `BlockOrEBB`, but that contains the `SlotNo` of the block and the `EpochNo` of the EBB. I don't think you can easily compute the `EpochNo` where this event is being traced. So feel free to introduce an `IsEBB`-ish data type (isomorphic to `Bool`).

Reply:
I pushed a commit with this message:

```
    consensus: add IsEBB flag type
    
    `IsEBB` is primarily intended for unnamed data constructor arguments. Functions
    and record fields with signatures matching `...IsEBB :: ... -> Bool` are clear
    enough as-is.
```

It adds a flag type `Ouroboros.Consensus.Block.IsEBB`.